src/databricks/labs/ucx/install.py

```diff
@@ -807,6 +807,24 @@ def latest_job_status(self) -> list[dict]:
                 continue
         return latest_status
 
+    def repair_run(self, workflow):
+        try:
+            job_id_list = [job_id for step, job_id in self._state.jobs.items() if workflow == step]
+            if job_id_list:
+                job_id = job_id_list[0]
+                job_runs = list(self._ws.jobs.list_runs(job_id=job_id, limit=1))
+                state = job_runs[0].state
```

Comment:
Bug: will crash when there are no runs

Reply:
Changed the code. If there is no runs it will throw a warning message